torch/_inductor/fx_passes/micro_pipeline_tp.py

```diff
@@ -292,6 +295,8 @@ class _Matmul:
     arg_ancestor_nodes: OrderedSet[torch.fx.Node] = field(init=False)
     A_node: torch.fx.Node
     B_node: torch.fx.Node
+    pre_mm_reshape: Optional[torch.fx.Node]
+    post_mm_reshape: Optional[torch.fx.Node]
```

Comment:
Can you educate me what `A_node` and `B_node` stand for?

Reply:
A_node and B_node are the nodes representing the input tensors to the matmul (i.e., `C = torch.mm(A, B)` or `C. = torch._scaled_mm(A, B, A_scale, B_scale)`).